libmscore/segment.cpp

```diff
@@ -895,6 +895,8 @@ bool Segment::setProperty(Pid propertyId, const QVariant& v)
                   break;
             case Pid::LEADING_SPACE:
                   setExtraLeadingSpace(v.value<Spatium>());
+                  for (Element* e : _elist)
+                        if(e) e->setGenerated(false);
```

Comment:
not crazy about the formatting.  If you do want to force this to one line, still have the space after the "if", but consider breaking it into lines as usual (and adding braces on the "for", even though not strictly necessary, as we do elsewhere).

Reply:
My mistake, old habit of mine. I’m aware that that’s against the style guide, I’ll fix it when I get back to my computer :)